tutorial.md

```diff
@@ -1,101 +1,85 @@
-Application to Trace" window. Type "gapid" in the filter text box to filter down
-applications to the ones that contain "gapid" in their package name. Expand the
-package, select "com.google.android.gapid.VkSampleActivity", and press "OK". This
-brings you back to the capture options window, with the "Application" field
-populated by the Application you selected.
+    <div class="info" markdown="span">	
+    AGI comes with a simple Vulkan application that we are going to use as an example.<br><br>
+    1. Click on the "..." button right next to the "Application" text entry to list applications that can be traced on the selected device.<br>
+    2. This pops-up the "Select an Application to Trace" window.<br>
+    3. Type "gapid" in the filter text box to filter down applications to the ones that contain "gapid" in their package name.<br>
+    4. Expand the package, select "com.google.android.gapid.VkSampleActivity", and press "OK".<br>
+    5. This brings you back to the capture options window, with the "Application" field populated by the Application you selected.<br>
+    </div>
 
-You may also choose another Vulkan application if you prefer.  If there is only
-one activity in the package, you can simply select the package instead of the Activity.
+    You may also choose another Vulkan application if you prefer.  If there is only one activity in the package, you can simply select the package instead of the Activity.
 
-In the "Application" section, you can leave the other fields empty.
+7. In the "Application" section, you can leave the other fields empty.
 
-Under "Start and Duration" section, set "Start at" to "Manual", and "Duration"
-to 2.
+8. Under "Start and Duration" section, set "Start at" to "Manual", and "Duration" to 2.
 
-In the "Trace Option" section, click on the "Configure" button to pop-up a
-configuration window where you can select which can of performance counters you
-want to be captured.
+9. In the "Trace Option" section, click on the "Configure" button to pop-up a configuration window where you can select which can of performance counters you want to be captured.
 
-<img src="../images/system_profiler/capture-config.png" width="500px">
+    <img src="../images/system_profiler/capture-config.png" width="500px">
 
-In the "Output" section, please select an "Output Directory" where the capture
-file will be stored. The "File Name" should be auto-filled, you can edit the
-file name if need be.
+10. In the "Output" section, please select an "Output Directory" where the capture file will be stored. The "File Name" should be auto-filled. You can also edit the file name.
 
-In the GPU section, click on "Select" to bring up the counter selection dialog.
-Click on "default" to select the set of default counters and press "OK".
+11. In the GPU section, click on "Select" to bring up the counter selection dialog. Click on "default" to select the set of default counters and press "OK".
```

Comment:
This should be before 10, since it's in the profiling dialog.

Reply:
See https://github.com/google/agi/pull/182/commits/62bf1b179b61c5e8ae25464df3961aa218ff1bd8